graphql_compiler/tests/test_compiler.py

```diff
@@ -10,7 +10,23 @@
 from .test_helpers import compare_gremlin, compare_input_metadata, compare_match, get_schema
 
 
-def check_test_data(test_case, test_data, expected_match, expected_gremlin):
+def check_test_data_match(test_case, test_data, expected, schema_based_type_equivalence_hints):
```

Comment:
I don't think we need the two separate functions, since we want to keep feature parity.